Index: amugeona/src/main/java/com/shashashark/amugeona/model/dto/ArticleDto.java
===================================================================
diff --git a/amugeona/src/main/java/com/shashashark/amugeona/model/dto/ArticleDto.java b/amugeona/src/main/java/com/shashashark/amugeona/model/dto/ArticleDto.java
new file mode 100644
--- /dev/null	
+++ b/amugeona/src/main/java/com/shashashark/amugeona/model/dto/ArticleDto.java	
@@ -0,0 +1,26 @@
+package com.shashashark.amugeona.model.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+import java.util.Date;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ArticleDto {
+    private Long articleSeq;
+    private Long userSeq;
+    private Long boardSeq;
+    private String title;
+    private String content;
+    private Double starRating;
+    private Integer starCnt;
+    private Date createdAt;
+    private Date modifiedAt;
+}
Index: amugeona/src/main/java/com/shashashark/amugeona/model/dto/ArticleUpdateParam.java
===================================================================
diff --git a/amugeona/src/main/java/com/shashashark/amugeona/model/dto/ArticleUpdateParam.java b/amugeona/src/main/java/com/shashashark/amugeona/model/dto/ArticleUpdateParam.java
new file mode 100644
--- /dev/null	
+++ b/amugeona/src/main/java/com/shashashark/amugeona/model/dto/ArticleUpdateParam.java	
@@ -0,0 +1,15 @@
+package com.shashashark.amugeona.model.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ArticleUpdateParam {
+    private String title;
+    private String content;
+}
Index: amugeona/src/main/java/com/shashashark/amugeona/model/service/ArticleService.java
===================================================================
diff --git a/amugeona/src/main/java/com/shashashark/amugeona/model/service/ArticleService.java b/amugeona/src/main/java/com/shashashark/amugeona/model/service/ArticleService.java
new file mode 100644
--- /dev/null	
+++ b/amugeona/src/main/java/com/shashashark/amugeona/model/service/ArticleService.java	
@@ -0,0 +1,44 @@
+package com.shashashark.amugeona.model.service;
+
+import com.shashashark.amugeona.model.dto.ArticleDto;
+import com.shashashark.amugeona.model.dto.ArticleUpdateParam;
+import com.shashashark.amugeona.model.entity.Article;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface ArticleService {
+
+    //글 작성하기
+    void writeArticle(ArticleDto articleDto);
+
+    //글 하나 불러오기
+    Optional<ArticleDto> selectOne(Long articleSeq);
+
+    //글 전체 불러오기
+    List<ArticleDto> selectAll();
+
+    //글 수정하기
+    void updateArticle(ArticleUpdateParam param);
+
+    //글 삭제하기
+    void deleteArticle(Long articleSeq);
+
+    //db에서 불러오기
+    default ArticleDto toDto(Article article){
+        return ArticleDto.builder()
+                .articleSeq(article.getArticleSeq())
+//                .userSeq(article.getUser().getUserSeq())
+                .boardSeq(article.getBoard().getBoardSeq())
+                .title(article.getTitle())
+                .starRating(article.getStarRating())
+                .starCnt(article.getStarCnt())
+                .createdAt(article.getCreatedAt())
+                .modifiedAt(article.getModifiedAt())
+                .build();
+    }
+
+    default  Article toEntity(ArticleDto articleDto){
+        return Article
+    }
+}
Index: amugeona/src/main/java/com/shashashark/amugeona/model/repository/ArticleRepository.java
===================================================================
diff --git a/amugeona/src/main/java/com/shashashark/amugeona/model/repository/ArticleRepository.java b/amugeona/src/main/java/com/shashashark/amugeona/model/repository/ArticleRepository.java
new file mode 100644
--- /dev/null	
+++ b/amugeona/src/main/java/com/shashashark/amugeona/model/repository/ArticleRepository.java	
@@ -0,0 +1,9 @@
+package com.shashashark.amugeona.model.repository;
+
+import com.shashashark.amugeona.model.entity.Article;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface ArticleRepository extends JpaRepository<Article, Long> {
+}
Index: amugeona/src/main/java/com/shashashark/amugeona/model/service/Impl/ArticleServiceImpl.java
===================================================================
diff --git a/amugeona/src/main/java/com/shashashark/amugeona/model/service/Impl/ArticleServiceImpl.java b/amugeona/src/main/java/com/shashashark/amugeona/model/service/Impl/ArticleServiceImpl.java
new file mode 100644
--- /dev/null	
+++ b/amugeona/src/main/java/com/shashashark/amugeona/model/service/Impl/ArticleServiceImpl.java	
@@ -0,0 +1,43 @@
+package com.shashashark.amugeona.model.service.Impl;
+
+import com.shashashark.amugeona.model.dto.ArticleDto;
+import com.shashashark.amugeona.model.dto.ArticleUpdateParam;
+import com.shashashark.amugeona.model.repository.ArticleRepository;
+import com.shashashark.amugeona.model.service.ArticleService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+@RequiredArgsConstructor
+public class ArticleServiceImpl implements ArticleService {
+
+    private final ArticleRepository articleRepository;
+
+    @Override
+    public void writeArticle(ArticleDto articleDto) {
+
+    }
+
+    @Override
+    public Optional<ArticleDto> selectOne(Long articleSeq) {
+        return Optional.empty();
+    }
+
+    @Override
+    public List<ArticleDto> selectAll() {
+        return null;
+    }
+
+    @Override
+    public void updateArticle(ArticleUpdateParam param) {
+
+    }
+
+    @Override
+    public void deleteArticle(Long articleSeq) {
+
+    }
+}
Index: amugeona/src/main/java/com/shashashark/amugeona/model/entity/Article.java
===================================================================
diff --git a/amugeona/src/main/java/com/shashashark/amugeona/model/entity/Article.java b/amugeona/src/main/java/com/shashashark/amugeona/model/entity/Article.java
new file mode 100644
--- /dev/null	
+++ b/amugeona/src/main/java/com/shashashark/amugeona/model/entity/Article.java	
@@ -0,0 +1,41 @@
+package com.shashashark.amugeona.model.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+public class Article {
+    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long articleSeq;
+
+//    @ManyToOne
+//    @JoinColumn(name = "user_seq")
+//    private User user;
+
+    @Column(name="board_seq")
+    private Long boardSeq;
+
+    @ManyToOne
+    @JoinColumn(name = "board_seq")
+    private Board board;
+
+    private String title;
+    private String content;
+    private Double starRating;
+    private Integer starCnt;
+
+    @Temporal(TemporalType.TIMESTAMP)
+    private Date createdAt;
+
+    @Temporal(TemporalType.TIMESTAMP)
+    private Date modifiedAt;
+}
